app/src/main/java/com/grupo5/mymovies/MainActivity.kt

```diff
@@ -1,40 +1,65 @@
 package com.grupo5.mymovies
 
-import androidx.appcompat.app.AppCompatActivity
+import android.opengl.Visibility
 import android.os.Bundle
 import android.util.Log
+import android.view.View
+import android.widget.ProgressBar
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.isVisible
+import androidx.recyclerview.widget.DefaultItemAnimator
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.grupo5.mymovies.adapters.MovieAdapter
 import com.grupo5.mymovies.model.MoviesService
 import kotlinx.coroutines.*
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 import kotlin.coroutines.CoroutineContext
+import com.grupo5.mymovies.model.Movie
+
 
 class MainActivity : AppCompatActivity(), CoroutineScope {
 
     override val coroutineContext: CoroutineContext
         get() = Dispatchers.Main + job
 
     private lateinit var job: Job
-
+    private lateinit var mAdapter : MovieAdapter
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        val recyclerView = findViewById<View>(R.id.moviesRecycleView) as RecyclerView
```

Comment:
You can directly access to the UI components using the view id.
For example `moviesRecycleViw.setHasFixedSize(true)`
So it's no longer necessary using the **findViewById**

Reply:
thanks! that has really improve since my java android times xD, will fix it

Reply:
fixed

Reply:
great! 👍